src/utils.ts

```diff
@@ -0,0 +1,50 @@
+import { Readable } from 'stream';
+import { Block } from './types';
+
+export function prependZeroes(number: number) {
+  let string_number = number.toString();
+  while (string_number.length < 12) {
+    string_number = "0" + string_number;
+  }
+  return string_number;
+}
+
+export async function parseBody<T>(stream: Readable): Promise<T> {
+  const contents = await streamToString(stream);
+  const parsed: T = JSON.parse(contents, renameUnderscoreFieldsToSnakeCase);
+  return parsed;
+}
+
+// the function got from
+// https://github.com/aws/aws-sdk-js-v3/issues/1877#issuecomment-755387549
+function streamToString (stream: Readable): Promise<string> {
```

Comment:
Should we reuse [the library](https://www.npmjs.com/package/stream-to-string)?

Reply:
What for? Just for shortening the code or do you see any additional benefits from using that library?

Reply:
Just not to implement things twice :)
Probably that's a js-codespace habit